integrationtest/src/main/scala/io/gearpump/integrationtest/minicluster/RestClient.scala

```diff
@@ -64,14 +77,45 @@ class RestClient(host: String, port: Int) {
     case ex: Throwable => null
   }
 
-  /*
-  def queryAppDetail(appId: Int): AppMasterSummary = {
-      val resp = callApi(s"appmaster/$appId?detail=true")
-      if (resp.startsWith("java.lang.Exception: Can not find Application:"))
-        null
-      else upickle.default.read[AppMasterSummary](resp)
+  def queryStreamingAppDetail(appId: Int): StreamAppMasterSummary = {
+    val resp = callApi(s"appmaster/$appId?detail=true")
+    if (resp.startsWith("java.lang.Exception: Can not find Application:"))
+      null
+    else upickle.default.read[StreamAppMasterSummary](resp)
+  }
+
+  def getExecutorInfos(appId: Int): List[ExecutorBrief] = {
+    val streamAppMasterSummary = queryStreamingAppDetail(appId)
+    if(streamAppMasterSummary != null) {
+      streamAppMasterSummary.executors
+    } else {
+      List.empty
     }
-    */
+  }
+
+  def killAppMaster(appId: Int): Boolean = {
```

Comment:
The PR was +1 and merged too early. `killAppMaster` and `killExecutor` need actor system, this is not REST approach. If we cannot kill corresponding appmaster/executor at process level, then we'd reconsider move these test cases to unit tests.